_posts/2018-11-16-stairway-to-health.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+authors: [gert-jan_schokkaert]
+title: "Visualising IoT data with tableau"
+image: /img/2018-11-16-stairway-to-health/stairway-to-health.png
+tags: [Development,Translations,Angular,i18n,Crowdin]
+category: Development
+comments: true
+---
+
+# VisionWorks meets JWorks: StairwayToHealth
+We from [VisionWorks][] were asked to rebuild the visualisation dashboard [JWorks][] used in the [application][] they built as a result of the internal StairwayToHealth project (you can find more information about that project [here][]). We decided to use [Tableau][], a popular BI Visualisation tool we largely use at our clients.
+We developed the dashboard working around some key questions while keeping the appearance of the dashboard in line with the dashboard JWorks developed.
+
+In the following section I will explain how the dashboard is currently set up and how to use it properly.
+Next I will go over the features we can add in the future to allow the user to go even deeper in their analysis.
+
+# Dashboard overview
+
+The dashboard is built to answer **the following questions**:
+
+1. What is the percentage of people taking the stair/elevator at Ordina **today**?
+2. How is the same metric on week/month/year basis? What is it in absolute numbers?
+3. Within each day/week/month/year base how does it evolve over time?
+4. And across days/weeks/months/years? Are people taking the stairs more this week compared to last week?
+
+To answer these questions, the dashboard consists of **three main parts**.
```

Comment:
The dashboard will try to provide answers to these questions using the following **three main parts**.